### ISSUE TITLE
Action pane UX: containers identification

### ISSUE THREAD

**Issue by pjjr** [apiaryio/attributes-kit#229]
In some cases, not even too complex, in data structure viz, it's hard to follow which line belongs to which container.
Some of these would be very helpful:
a. **collapse all** button at the top
b. different background or vertical line **color** for odd/even container (1%~2% darker)
 - background color is hard to imagine; losing of the simple flat design is in risk

![image](https://cloud.githubusercontent.com/assets/17746143/14171817/389a5cd0-f735-11e5-975c-799499d87424.png)

**Comment by tjerabek** [comment]
Thanks for report @pjjr We work on better rendering of Attributes Kit. If you are interested to help us, please send me your MSON to tomas@apiary.io. I'd really appreciate your feedback and help!
